wit/blobstore.wit

```diff
@@ -1,5 +1,3 @@
-package wasi:blobstore;
```

Comment:
Shouldn't all of these be `package wasi:blobstore@0.1.0` and not removed?

Reply:
I think one declaration of package is sufficient, no?

Reply:
I thought it was per file, but then I ran into other examples where it wasn't the case.

Reply:
Cool it seems like we are aligned here. Could you please give it another review and so I can merge it once it's approved.